ansible/inventory.yml

```diff
@@ -0,0 +1,13 @@
+all:
+  children:
+    iot:
+      hosts:
+        debian10:
+          ansible_host: 192.168.88.21
+          vpn_user: acq02
```

Comment:
The idea is inventory file is stored locally, it's not been committed to any remote repository. The file I put here is just sample form for future deployments using this playbook to follow.
How do you think about it?

Reply:
Yes, `inventory` will be store local for test, and will be generated by programming.
But, I don't want to add any information about any application (here is VPN). Coz, `inventory` in my term/convention will store only connection information of deployment target.
It will be used to provision for many app, not only `vpn`